src/Mapper/Mapper.test.ts

```diff
@@ -149,6 +150,73 @@ describe("Mapper", () => {
       fakeModel.fields.fakeItems.serialize = false;
     });
 
+    describe("transformPatch", () => {
+      let originalModel;
+      let originalModelSpy;
+      let originalFullText;
+
+      beforeEach(() => {
+        originalFullText = lorem.slug();
+        originalModel = new MockModel({ ...mockModelData, fullText: originalFullText });
+
+        originalModelSpy = spy();
+        Object.defineProperty(fakeModel, "original", {
+          get() {
+            originalModelSpy();
+            return originalModel;
+          },
+          configurable: true,
+        });
+      });
+
+      it("calls transform on the model and the model.original", async () => {
+        const transformStub = stub(mapper, "transform");
+
+        await mapper.transformPatch(fakeModel);
+
+        expect(transformStub.callCount).to.eq(2);
+
+        expect(transformStub.calledWithExactly(fakeModel)).to.be.true;
+        expect(transformStub.calledWithExactly(originalModel)).to.be.true;
```

Comment:
I wonder if there is way to write this test that doesn't expect 3 different things (generally we want to limit it to one per test). I think `transformStub.args` will also be helpful as you could do something like `expect(transformStub.args).to.have.lengthOf(3).and.deep.equal([fakeModel, originalModel])`

Reply:
I'll change it but that really seems like a loophole to me.  Technically the word `expect` is only written one time but multiple things are being asserted because of the `and` chain.

Reply:
Actually spy.args returns an array of an array and if we flattened it just for readability we would be destroying useful data like if we called transform(one, two) and then transform().  The test would pass but the call would be wrong.